docs/modules/user-guide/partials/proc_devfile-properties.adoc

```diff
@@ -0,0 +1,258 @@
+[id="proc_devfile-properties_{context}"]
+= Devfile properties 
+
+[role="_abstract"]
+Use this page to work with yaml files of devfile properties. See the following table of contents to go to the devfile property you want to work with:
+
+== SchemaVersion
+
+[source,yaml]
+----
+v2.0
+---
+schemaVersion: 2.0.0
+----
+
+[cols="1,1"]
+|===
+|Key
+|schemaVersion
+
+|Type
+|string
+
+|Required
+|yes
+
+|Description
+|v2.0.0 of the schema
+|===
```

Comment:
Maybe it would make more sense to put all the information in one table at the top? Since they all share the same categories (key, type, required, and description) and tables should help the user order more complex information in a nice overview. Having these small tables with only one row all over the guide seems to take a lot of space?
Maybe you'd be interested in this article about using tables!
https://medium.com/@heyoka/lets-bring-table-to-the-table-again-f1ae751159d5

Reply:
Thanks, for the article! The Brady Bunch reference is fire! I'll think about the one table...I might wait to get the dev opinion on one shared table or separate!